web/src/pages/Status.jsx

```diff
@@ -290,7 +301,7 @@ export function Status() {
         sx={{ left: -55 }}
       >
         {[0, 1, 2, 3].map((idx) => {
-          const checked = iFilter[idx];
+          const checked = getThreatImpactOn(idx + 1);
```

Comment:
本当は `{[0, 1, 2, 3].map((idx) => {` ではなく、　threat_impact の文字列表現 (imemdiate, offcycle...)でmapした方が読み手がわかりやすい気がします
`{Object.values(threatImpactName).map(impactName => {`


数値でmapを続ける場合は以下
```suggestion
          const impactName = threatImpactName[idx + 1]
          const checked = impactFilters.includes(impactName])
```

Reply:
指摘通り、URLのparamをimpactFilterの配列に変更しました。

> 本当は {[0, 1, 2, 3].map((idx) => { ではなく、　threat_impact の文字列表現 (imemdiate, offcycle...)でmapした方が読み手がわかりやすい気がします
こちらは、少し影響範囲が広がり、今スプリント中に終わらない見込みのため、対応していません。